cli/dcoscli/task/main.py

```diff
@@ -550,3 +562,33 @@ def _load_slaves_state(slaves):
                 DefaultError('Error accessing slave: {0}'.format(e)))
 
     return reachable_slaves
+
+
+def _metrics(summary, task_id, json_):
+    """
+    Get metrics from the specified task.
+
+    :param summary: summarise output if true, output all if false
+    :type summary: bool
+    :param task_id: mesos task id
+    :type task_id: str
+    :param json: print raw JSON
+    :type json: bool
+    :return: Process status
+    :rtype: int
+    """
+
+    master = mesos.get_master()
+    slave_id = master.task(task_id)['slave_id']
```

Comment:
I've added a guard for this, but I don't think a Mesos task object can ever not have a slave_id field.